stylesheets/styles.css

```
@@ -26,7 +26,7 @@ time, mark, audio, video {
 /* Base text styles */
 
 body {
-  padding:10px 50px 0 0;
+  padding:10px 50px 20px 20px;
```

But why is there such an asymmetric padding? Can we try with this:
```suggestion
  padding:50px 50px 20px 20px;
```

---

@fingolfin I've pushed an update now. In my opinion, the space at the top of the page feels very big now. What do you say?